src/useFirebaseUser.js

```diff
@@ -87,6 +88,7 @@ const useFirebaseUser = () => {
     user, // unmodified Firebase user, undefined if not authed
     claims: customClaims,
     initialized,
+    authCookieComplete,
```

Comment:
@hershmire - Were you going to update withAuthUser to use this?

Reply:
I was going to work on a fix when I had time. It's been a really busy couple months for me so I haven't been able to get to this yet.

Reply:
Ah ok. That makes sense I was wondering how this worked.